### ISSUE TITLE
have someone met this issue?

### ISSUE THREAD

**Issue by zHanami** [endernewton/tf-faster-rcnn#267]
![image](https://user-images.githubusercontent.com/27133144/34137793-9094a786-e4a7-11e7-874c-52d572e409ac.png)
all thing seems go well ,but when it carried out the 30000/70000 iters it came,may caused by saver?
![image](https://user-images.githubusercontent.com/27133144/34137844-d59be3d0-e4a7-11e7-97ca-88095b9db6db.png)

**Comment by endernewton** [comment]
did it actually save anything?

**Comment by zHanami** [comment]
yeah, it can save the ckpt and can continue to train when i call instruction， but it break down 2 times due to the same issue, i do not why. however, the test result is good ,reaching to 70.8.
And i want to know if the defult code for voc2007 is just modified in 3 parts, which has  mentioned in your paper, and no more hyperparameters have you changed from the original faster-rcnn code for tensorflow?
